src/actions/helpers/getPlayersHelper.js

```diff
@@ -0,0 +1,28 @@
+import socket from '../../socketHandlers/index';
+
+const SOCKET_EVENT_TYPES = {
+  EMIT: 'getPlayers',
+  ON: 'resultGetPlayers'
+}
+
+const getPlayersPromise = (roomIds) => {
+  return roomIds.map((roomId) => {
+    return new Promise((resolve, reject) => {
+      socket.on(SOCKET_EVENT_TYPES.ON + roomId, (data) => {
```

Comment:
I think this is not good implementation. event type should be constant value but this event is not fixed value. (Depends on roomId.)

Reply:
@duyoji 
Why should it be a constant value?